.changeset/fluffy-kiwis-sell.md

```diff
@@ -0,0 +1,10 @@
+---
+"cojson-core-wasm": minor
+"jazz-tools": minor
+"cojson": minor
```

Comment:
```suggestion
"cojson-core-wasm": patch
"jazz-tools": patch
"cojson": patch
```

We ship all non-breaking changes as patch, since we are still in v0

Reply:
Ok :+1: